### ISSUE TITLE
Designer not launching

### ISSUE THREAD

**Issue by inoxas78** [lichr/zircon-ha-proxy#9]
Blank screen when launch designer.

HAOS on x86x64 machine.

Any idea?

**Comment by shavermcspud** [comment]
Same here.

**Comment by bkabela** [comment]
Same problem here, from the console I get the following error:
```
Failed to load resource: the server responded with a status of 500 (Internal Server Error)
```

Going to http://IP:11200/online/api/pub/methods/load_designer?group=*****&project=*****

I get the following:

```
Error: Group part not found: user
    at ProjectCore.getGroupPart (/app/proxy/services/proxy-core/project-core.js:156:19)
    at async ProjectCore.loadDesigner (/app/proxy/services/proxy-core/project-core.js:170:19)
    at async /app/proxy/routes/use-online.js:17:26
```

**Comment by chris-xinhai-li** [comment]
Thank the for the detail, now I can replicate the problem in my environment, looks like new project + local only will cause it.

Will fix it as soon as possible.

**Comment by chris-xinhai-li** [comment]
The fix has been release as 0.3.2, please upgrade your add-on. After that, your current project won't be fixed, please create a new one and it should work.

**Comment by bkabela** [comment]
That fixed it for me, thanks!

**Comment by chris-xinhai-li** [comment]
fixed in 0.3.1
